invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme.less

```diff
@@ -272,12 +272,23 @@ body {
 
 // Dashboard user information header
 
-.ui.container {
-  &.dashboard-user-header-container {
-    margin-bottom: 20px;
-  }
+.dashboard-header {
+  background: #F8F8F9;
+  margin-top: -20px;
+  padding-top: 20px;
+  padding-bottom: 5px;
 }
 
+.ui.image{
+  &.tinier {
+    width: 54px !important;
+    height: auto !important;
+  }
+
+  &.circle {
+    border-radius: 50% !important;
```

Comment:
is this needed? because in the doc, it should be circular: https://semantic-ui.com/elements/image.html#circular

Reply:
Circular from semantic ui does not create a proper circle, it makes borders kinda circular.
For creating a real circle I had to define that property.
Differences between semantic's circular and this circle can be seen comparing the screenshot in the issue with the one in this PR respectively.

Reply:
It looks very circular to me in the doc! Can we see this together?